docs/concepts/tools/kubectl/object-management-overview.md

```diff
@@ -170,20 +159,16 @@ Disadvantages compared to imperative object configuration:
 - Declarative object configuration is harder to debug and understand results when they are unexpected.
 - Partial updates using diffs create complex merge and patch operations.
 
-{% comment %}
-For a tutorial on how to use Yaml Config with multiple writers, see:
-[App Management Yaml Config](/docs/tutorials/kubectl/app-management-using-yaml-config-multiple-writers/)
-{% endcomment %}
-
 {% endcapture %}
 
 {% capture whatsnext %}
+- [Managing Kubernetes Objects Using Imperative Commands](/docs/concepts/tools/kubectl/object-management-using-commands/)
```

Comment:
Links are broken. Should add `.md` at the end. Same thing happens in other files. Please check.